tests/example_app/example/about/our_story/bp_story.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint, render_template
+
+bp_aboutStory = Blueprint("our_story", __name__,
+                          template_folder="./templates")
+
+
+@bp_aboutStory.route("/our-story")
+def pageAboutStory():
```

Comment:
Can you rename this to `def story` with a URL of `/story` and also rename the HTML file to `story.html`?